monai/networks/nets/__init__.py

```diff
@@ -38,10 +38,11 @@
     EfficientNet,
     EfficientNetBN,
     EfficientNetBNFeatures,
+    EfficientNetEncoder,
     drop_connect,
     get_efficientnet_image_size,
 )
-from .flexible_unet import FlexibleUNet
+from .flexible_unet import BACKBONE, FlexibleUNet, Register
```

Comment:
`monai.networks.nets.Register` sounds very general, but the actual implementation is only for the FlexibleUNet... this might be an issue cc @Nic-Ma

Reply:
Hi @wyli ,

I didn't put comment for this one because it's within the `flexible_unet.py` file, but I agree a more specific name would be better.

Thanks.

Reply:
Maybe we can call it `EncoderRegister` or some other name?

Thanks.

Reply:
ok, or removing `from .flexible_unet import BACKBONE, Register` is also fine?

Reply:
I would suggest: EncoderRegister or make a alias for the `FlexibleUNet` vs `FlexUNet`?
@wyli What do you think?

Thanks.

Reply:
And `FLEXUNET_BACKBONE`。

Reply:
Hi @Nic-Ma ,
      I have added the alias and changed the backbone name.
Thanks.